.github/workflows/shuttle.yml

```diff
@@ -40,11 +37,7 @@ jobs:
       - name: Build
         run: cargo build --locked --verbose
       - name: Deploy
-        if: |
```

Comment:
If we are removing this check then it would make sense to remove the `dorny/paths-filter` step as well. The idea was that I only want to run the shuttle deployments when there is a change made to `Cargo.toml`.

Reply:
ok, then I'll remove it, unless you can think of another way.

Reply:
done